others/sub_sequences.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 by Contributors
+#include <cmath>
+#include <iostream.h>
+ 
+void SubSequences(char *in, int i, char *out, int j) {
```

Comment:
use camel casing for function name such as `subSequences` in this case

Reply:
and please provide a berif idea of what operation this function is performing so that it will be easy for the reader to understand what you have done